.gitmodules

```diff
@@ -18,7 +18,7 @@
 	url = https://github.com/arangodb/linenoise-ng
 [submodule "libraries/cmake/source/lzma/src"]
 	path = libraries/cmake/source/lzma/src
-	url = https://github.com/xz-mirror/xz
+	url = https://github.com/tukaani-project/xz.git
```

Comment:
I assume this is a legit org?

Reply:
Yes, it was already in our libraries manifest: https://github.com/osquery/osquery/blob/master/libraries/third_party_libraries_manifest.json, also if you check the about of the old xz-mirror they have archived it and moved to that other repo.